experiments/CamemBERT/model/CamemBERT.py

```diff
@@ -44,7 +45,10 @@ def forward(self, input):
 
 print("Parsing sentence tokens.")
 example_input = prepare_sentence_tokens(model_name, sentence)
-print(example_input.shape)
+print("example_input shape: ", example_input.shape)

```

Comment:
Can you add comment above this line to show what is the original shape?

Reply:
sure